.github/workflows/crawl.yml

```diff
@@ -0,0 +1,55 @@
+---
+name: Dashboard crawl (daily)
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '15 4 * * *'
+
+jobs:
+  dashboard-crawl-staging:
+    name: dashboard crawl (staging)
+    environment: staging
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: crawl download
+        uses: usds/cloud-gov-cli@master
+        with:
+          command: run-task dashboard-stage --command "php public/index.php campaign status omb-monitored download"
```

Comment:
Does the CLI version we're using support the --wait? We might not to run multiple crawls at the same time and instead run them serially.